extensions/metadata_exchange/config.pb.html

```diff
@@ -1,7 +1,7 @@
 ---
 title: Metadata Exchange Config
 description: Configuration for Metadata Exchange Filter.
-location: https://istio.io/docs/reference/config/proxy_extensions/metadata_exchange.html
+location: https://istio.io/docs/reference/config/proxy_extensions/metadata_exchange.html.
```

Comment:
is this changed added manually or updated by make extensions-docs?

Reply:
updated using make extension-docs.. I can revert this change?